src/libs/getEmailKeyboardType/index.js

```diff
@@ -0,0 +1,8 @@
+
+/**
+ * * Return the default keyboard type when running on Web/Desktop
```

Comment:
There's still a leading `*` that shouldn't be there:

```suggestion
 * Return the default keyboard type when running on Web/Desktop
```

Reply:
@HorusGoul  - Done